server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -393,6 +393,15 @@ private void registerAsyncMetrics(MeterRegistry registry) {
             )
         );
 
+        metrics.add(
+            registry.registerLongGauge(
+                "es.node.stats.http.current_open",
+                "Current number of open HTTP connections for the node",
+                "connections",
+                () -> new LongWithAttributes(stats.getOrRefresh().getHttp().getServerOpen())
+            )
+        );
```

Comment:
do we need a test for the new metric?

Reply:
Yes, plus it's failing CI and I suspect is for a good reason, I'll go back to it add tests and investigate CI failures.